libraries/botbuilder-core/src/activityHandler.ts

```diff
@@ -414,11 +414,17 @@ export class ActivityHandler extends ActivityHandlerBase {
     */
     protected async onInvokeActivity(context: TurnContext): Promise<InvokeResponse> {
         try {
-            if (context.activity.name && (context.activity.name === verifyStateOperationName || context.activity.name === tokenExchangeOperationName)) {
-                await this.onSignInInvoke(context);
-                return { status: StatusCodes.OK };
+            switch (context.activity.name) {
+                case verifyStateOperationName:
+                case tokenExchangeOperationName: {
+                    await this.onSignInInvoke(context);
+                    return { status: StatusCodes.OK };
+                }
+                case "healthCheck":
```

Comment:
>healthCheck [](start = 22, length = 11)

Should this be a constant as well?

Reply:
we are missing constants for most of the invokes - for example all the Teams invokes - we have them for some of the security stuff - the thought was we would just do them all together. Currently this is also pretty internal and not primary application programming model so we weren't super concerned

Reply:
This point was brought up in C# as well: https://github.com/microsoft/botbuilder-dotnet/pull/3796/files#r414133232

This could be put into a constant, but it's expected that it's mainly used by our libraries, not the developer's code.

Cleanup for internal (in this case, dispatching) constants could be done in another milestone.